content/en/docs/about/membership.md

```diff
@@ -0,0 +1,193 @@
+
+Responsibilities for most roles are scoped to these repositories.
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| Member | Active contributor in the community | Sponsored by 2 reviewers and multiple contributions to the project | Kubeflow GitHub org member|
+| Reviewer | Review contributions from other members | History of review and authorship in a repository | [OWNERS] file reviewer entry |
+| Approver | Contributions acceptance approval| Highly experienced active reviewer and contributor to a repository | [OWNERS] file approver entry|
+| WG Lead  | Provides technical leadership for a Working Group | Have sufficient domain knowledge to provide effective technical leadership | [wgs.yaml] entry |
+| WG Chair | Provides overall leadership for a Working Group | Have sufficient domain knowledge to provide effective leadership | [wgs.yaml] entry |
+
+**Note:** Detail documentation for Working Group structure and responsibilities can be found at [wg-governance.md](https://github.com/kubeflow/community/blob/master/wgs/wg-governance.md)
+
+## New contributors
+
+[New contributors] should be welcomed to the community by existing members, helped with PR workflow, and directed to relevant documentation and communication channels.
+
+## Established community members
+
+Established community members are expected to demonstrate their adherence to the principles in this document, familiarity with project organization, roles, policies, procedures, conventions, etc., and technical and/or writing ability. Role-specific expectations, responsibilities, and requirements are enumerated below.
+
+## Member
+
+Members are *[continuously active]* contributors in the community. They can have issues and PRs assigned to them and tests are automatically run for their PRs. Members are expected to remain active contributors to the community.
+
+**Defined by:** Member of the Kubeflow GitHub organization
+
+### Requirements
+
+- Enabled two-factor authentication on their GitHub account
+- Have made **at least** 2-3 [contributions] to the project or community.
+- Have read the [contributor guide]
+- Sponsored by 2 reviewers. **Note the following requirements for sponsors**:
+- **[Open an issue][membership request] against the kubeflow/internal-acls repo**
+  - Ensure your sponsors are @mentioned on the issue
+- **Open a pull request against the kubeflow/internal-acls repo**
+  - Complete every item on the checklist ([preview the current version of the template][membership template])
+  - Make sure that the list of contributions included is representative of your work on the project.
+- Have your sponsoring reviewers reply confirmation of sponsorship
+- Once your sponsors have responded, your request will be reviewed by the Kubeflow team. Any missing information will be requested
+
+### Responsibilities
+
+- Responsive to issues and PRs assigned to them
```

Comment:
Maybe we should also say, being active in the Kubeflow community, joining working groups meetings, Slack discussions, and project discussions.